var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -54,6 +63,15 @@ def cmake_args(self):
         else:
             options.append('-DENABLE_CUDA=OFF')
 
+        if '+hip' in spec:
+            arch = self.spec.variants['amdgpu_target'].value
+            options.extend([
+                '-DENABLE_HIP=ON',
+                '-DHIP_ROOT_DIR={0}'.format(spec['hip'].prefix),
+                '-DHIP_HCC_FLAGS=--amdgpu-target={0}'.format(arch)])
```

Comment:
I think you should use either `HIP_HIPCC_FLAGS` or `HIP_CLANG_FLAGS`. HCC is deprecated.

Reply:
I fixed it here and in camp, umpire, raja. The option HIP_HIPCC_FLAGS appears in a few other packages that I did not create, but I won't touch that during this pull request

Reply:
There isn't such a thing as `HIP_HIPCC_FLAGS` or `HIP_CLANG_FLAGS` in `hipcc`